Source/items.h

```diff
@@ -357,6 +357,16 @@ struct Item {
 		}
 	}
 
+	[[nodiscard]] bool IsScroll() const
+	{
+		return _iMiscId == IMISC_SCROLL || _iMiscId == IMISC_SCROLLT;

```

Comment:
Please use the `IsAnyOf` method.

Reply:
`IsAnyOf` seems to be a bit of an overkill in this case.

Reply:
It's not a matter of being overkill, but of consistency.

Seems like @AJenbo himself marked as resolved... I don't get it :(

Reply:
Found it not important/optional, wanted a better overview